packages/react-scripts/template/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '*.jpg'
+
+declare module '*.png'
+
+declare module '*.svg' {
+  import * as React from 'react'
```

Comment:
To be consistent with `index.js` this should be `import React from 'react'`. All this does assume that `esModuleInterop: true` is specified in tsconfig.json but I think this is best practice for new projects as it means that typescript and babel are more closely aligned.